morango/models.py

```diff
@@ -238,13 +245,19 @@ class SyncableModel(UUIDModelMixin):
     their data syncable across devices.
     """
 
+    # constant value to insert into partition strings in place of current model's ID, as needed (to avoid circularity)
+    ID_PLACEHOLDER = "<id>"
```

Comment:
Let's actually replace this with `${id}`, so it follows the same `Template` format as `*_scope_def` strings.